src/Componants/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+const SearchResults = props => {
+  return (
+    <table class="table table-hover table-dak">
+      <thead>
+        <tr>
+          <th scope="col">ID</th>
+          <th scope="col">Title</th>
+          <th scope="col">First name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room ID</th>
+          <th scope="col">Check in date</th>
+          <th scope="col">Check out date</th>
+          <th scope="col">Number of nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.fakeBookings.map(bookingInfo => {
```

Comment:
following the advice above to rename this prop. In here we could have something like:

```js
{props.bookings.map(booking => {
```